### DISCUSSION TITLE
Applying Privacy Preserving Techniques in NLP tasks

### DISCUSSION THREAD

**Discussion by majauhar** [OpenMined/courses#209] [Course: Our Privacy Opportunity] [answered]
Hi @yemikifouly! Could you suggest ways to apply privacy preserving techniques discussed in the course on NLP tasks such as sentiment analysis, translation, summarization, etc. Thanks 😃

**Comment by carlodavid012** [comment, marked as answer]
hi @majauhar you can check out this videos from Openmined PriCon on privacy preserving NLP. 
1. [Privacy-preserving Natural Language Processing - Fatemehsadat Mireshghallah](https://www.youtube.com/watch?v=jY-_34GJ3Ac)
2. [Tutorial: SyferText 0.1.0: Privacy-preserving NLP Over PyGrid - Alan Aboudib](https://www.youtube.com/watch?v=bRQdZM1B5QU)

**Reply by majauhar** [reply]
Thanks for the resource links.

**Comment by hershd23** [comment]
@majauhar we at Syfertext are actually working on the problems that you have written about. You can join #lib_syfertext channel if you're interested to contribute.

**Reply by majauhar** [reply]
that's cool. I just joined the channel. Thanks @hershd23
